raiden/tests/integration/test_echo_node.py

```diff
@@ -0,0 +1,197 @@
+# -*- coding: utf-8 -*-
+import pytest
+import gevent
+
+from raiden.utils.echo_node import EchoNode
+from raiden.api.python import RaidenAPI
+from raiden.tests.utils.network import CHAIN
+from raiden.tests.utils import get_channel_events_for_token
+
+
+# `RaidenAPI.get_channel_events` is not supported in tester
+@pytest.mark.parametrize('blockchain_type', ['geth'])
+@pytest.mark.parametrize('number_of_nodes', [4])
+@pytest.mark.parametrize('number_of_tokens', [1])
+@pytest.mark.parametrize('channels_per_node', [CHAIN])
+@pytest.mark.parametrize('reveal_timeout', [18])
+@pytest.mark.parametrize('settle_timeout', [64])
+def test_event_transfer_received_success(
+    token_addresses,
+    raiden_chain,
+):
+    app0, app1, app2, receiver_app = raiden_chain
+    token_address = token_addresses[0]
+    start_block = receiver_app.raiden.get_block_number()
+
+    expected = dict()
+
+    for num, app in enumerate([app0, app1, app2]):
+        amount = 1 + num
+        transfer_event = RaidenAPI(app.raiden).transfer_async(
+            token_address,
+            amount,
+            receiver_app.raiden.address,
+        )
+        transfer_event.wait(timeout=20)
+        expected[app.raiden.address] = amount
+
+    initiators = list()
+    received = list()
+    events = get_channel_events_for_token(receiver_app, token_address, start_block)
+    for event in events:
+        if event['_event_type'] == 'EventTransferReceivedSuccess':
+            received.append(event)
+            initiators.append(event['initiator'])
+
+    assert len(received) == 3
+    assert len(initiators) == 3
+    without_receiver_app = [app0.raiden.address, app1.raiden.address, app2.raiden.address]
+    assert set(without_receiver_app) == set(initiators)
+    for event in received:
+        assert expected[event['initiator']] == event['amount']
+
+
+# `RaidenAPI.get_channel_events` is not supported in tester
+@pytest.mark.parametrize('blockchain_type', ['geth'])
+@pytest.mark.parametrize('number_of_nodes', [4])
+@pytest.mark.parametrize('number_of_tokens', [1])
+@pytest.mark.parametrize('channels_per_node', [CHAIN])
+@pytest.mark.parametrize('reveal_timeout', [18])
+@pytest.mark.parametrize('settle_timeout', [64])
+def test_echo_node_response(
```

Comment:
If this fits in one line can you put the arguments in one line? Same for the following tests. By fitting I mean the <99 chars rule we have.